src/commands/createStaticWebApp/addLocationTelemetry.ts

```diff
@@ -5,14 +5,16 @@
 
 import { IStaticWebAppWizardContext } from "./IStaticWebAppWizardContext";
 
-export function addLocationTelemetry(wizardContext: IStaticWebAppWizardContext, key: 'appLocation' | 'apiLocation' | 'outputLocation', defaultValue: string): void {
+export function addLocationTelemetry(wizardContext: IStaticWebAppWizardContext, key: 'appLocation' | 'apiLocation' | 'outputLocation', defaultValue: string, valueFromSetting?: string): void {
     const value: string | undefined = wizardContext[key];
     let telemValue: string;
     if (value === undefined) {
         // no telemetry to add yet
         return;
     } else if (value.length === 0) {
         telemValue = 'empty';
+    } else if (value === valueFromSetting) {
+        telemValue = 'setting';
```

Comment:
I think there's more information to be gained here if the setting info was part of it's own properties. I might add properties like 'appLocationHasSetting' and 'appLocationMatchesSetting'

It would help you answer questions like these:
- If the user had a setting, did they use it or ignore it?
- Does the user's setting match the default value?